src/pudl/transform/classes.py

```diff
@@ -221,17 +249,10 @@ def normalize_strings(col: pd.Series, params: StringNormalization) -> pd.Series:
     * Translate to lower case.
     * Strip leading and trailing whitespace.
     * Consolidate multiple internal whitespace characters into a single space.
-    * Remove selected rare confounding punctuation marks.
-
-    This transform function has no associated TransformParams since it is simply a
-    dictionary with column names as keys and booleans as the values.
 
     Args:
         col: series of strings to normalize.
-        nullable: Whether to leave the series nullable or cast it back to a traditional
-            python string, with pd.NA replaced with the empty string.
-        params: Whether to do a transform at all. Included to keep the call signature
-            of the function the same as the other column transformations.
+        params: settings that determine string normalization behavior.
```

Comment:
I've updated this, but I don't think we should try and describe what the parameters contain in the functions where they're being used, since they're being described in detail in the classes where they're defined, which are right next to the functions and linked from the call signature. It's only possible in this case because the parameters are so simple.

Reply:
yeah i agree overall.